src/duckdb.js

```diff
@@ -3,6 +3,7 @@ import {getArrowTableSchema, isArrowTable, loadArrow} from "./arrow.js";
 import {duckdb} from "./dependencies.js";
 import {FileAttachment} from "./fileAttachment.js";
 import {cdn} from "./require.js";
+import {untyped} from "./table.js";
```

Comment:
This is a circular dependency since table.js imports DuckDBClient. We should define the `untyped` symbol here in duckdb.js, since it’s part of the (private) DuckDBClient API.

Reply:
I tried refactoring so that we don't need `untyped` in table.js (so it could just be defined in duckdb.js), but that did not seem straightforward to me. As an alternative, what about defining it somewhere that both table.js and duckdb.js could import it, like in fileAttachment.js?

Reply:
Spoke too soon! I think I see how to refactor. Will push an update soon.